### PR TITLE
Update parser property tests to latest changes in trees.

### PR DESCRIPTION
**Pull request by olafurpg**
I was nervous to find out if the recent fixes in the parser introduced regression. We stopped running these tests on every PR after moving to Travis CI. Turns out there are no regressions, and some bugs have been fixed along the way 🎉

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -71,6 +71,7 @@ testOnlyJVM := {
   val runBenchmarkTests = test.in(benchmarks, Test).value
   val runContribTests = test.in(contribJVM, Test).value
   val runTests = test.in(testsJVM, Test).value
+  val propertyTests = compile.in(testkit, Test).value
```
Comment (olafurpg) [inline build.sbt]: Just checking that these compile, as long as they compile we should be able to use git bisect to track down regressions in case something goes wrong.

**Comment by olafurpg** [comment]
The test failure is bogus, I've noticed that the --validate-links check is failing more often than usual lately.

File: scalameta/testkit/src/test/scala/scala/meta/testkit/ScalametaParserProperties.scala [hunk 2]
```diff
@@ -78,7 +78,7 @@ object ScalametaParserPropertyTest extends FunSuiteLike {
     val prettyPrinterBroken = result.count(_._2.kind == PrettyPrinterBroken)
     println(s"""Parser broken: $parserProken
                |Pretty printer broken: $prettyPrinterBroken""".stripMargin)
-    assert(parserProken <= 7)
-    assert(prettyPrinterBroken <= 1922)
+    assert(parserProken <= 4)
```
Comment (xeno-by) [inline scalameta/testkit/src/test/scala/scala/meta/testkit/ScalametaParserProperties.scala]: `Proken`? :)

Comment (olafurpg) [inline scalameta/testkit/src/test/scala/scala/meta/testkit/ScalametaParserProperties.scala]: Fixed, although this change is unrelated to the PR.
